prow/plugins/config.go

```diff
@@ -1940,11 +1940,19 @@ func (b *BranchCleaner) IsPreservedBranch(org, repo, branch string) bool {
 		if branch == pb {
 			return true
 		}
+
+		if match, _ := regexp.MatchString(pb, branch); match {
```

Comment:
nit: It would be preferable to log the error at the debug level rather than ignoring it.